runtime/doc/builtin.txt

```diff
@@ -796,6 +796,8 @@ complete_info([{what}])                                        *complete_info()*
 				dictionary containing the entries "word",
 				"abbr", "menu", "kind", "info" and "user_data".
 				See |complete-items|.
+		   shown	Indices of completion matches that are shown
```

Comment:
in parallel with the existing `pum_visible` and `pumvisible()`, seems like "visible" is the right name for this.

```suggestion
		   visible	Indices of completion matches that are shown
```